code/datums/observation/debrain.dm

```diff
@@ -0,0 +1,13 @@
+/**
+ * Observer Pattern Implementation: Debrained
+ *
+ * Raised when: A brainmob is created by the removal of a brain.
+ *
+ * Arguments that the called proc should expect:
+ *     /mob/living/brainmob: The brainmob that was created.
+ *     /obj/item/organ/internal/brain: The brain that was removed.
+ *     /mob/living/owner: The mob the brain was formerly installed in.
+ */
+/decl/observ/debrain
```

Comment:
i have no clue which ones are used on downstreams, iirc several are used on hearth? and psy was very concerned about removing hooks because outreach uses them
```
/hook/startup/proc/tgs_on_world_start()
	world.TgsNew()
	return TRUE

/hook/global_init/proc/tgs_on_init()
	world.TgsInitializationComplete()
	return TRUE

/hook/shutdown/proc/tgs_on_shutdown()
	world.TgsReboot()
	return TRUE
```
as an example of just the tgs ones